src/registrar/models/domain_application.py

```diff
@@ -654,11 +654,13 @@ def submit(self):
         self.submission_date = timezone.now().date()
         self.save()
 
-        self._send_status_update_email(
-            "submission confirmation",
-            "emails/submission_confirmation.txt",
-            "emails/submission_confirmation_subject.txt",
-        )
+        # Limit email notifications to transitions from Started and Withdrawn
+        if self.status == self.ApplicationStatus.STARTED or self.status == self.ApplicationStatus.WITHDRAWN:
```

Comment:
```suggestion
        limited_statuses = [self.ApplicationStatus.STARTED, self.ApplicationStatus.WITHDRAWN]
        # Limit email notifications to transitions from Started and Withdrawn
        if self.status in limited_statuses:
```
(Optional) While I doubt it, its possible that this list may be expanded in the future so storing it in an array like this makes it easier to expand